.eslintrc.js

```diff
@@ -102,5 +102,88 @@ module.exports = {
         'valid-typeof': 'off',
       },
     },
-  ],
-}
+    {
+      files: [
+        "lib/index-bundle.ts",
+        "lib/index.ts",
+        "lib/network/gephiParser.ts",
+        "test/NodesHandler.test.ts",
+        "test/dot-parser/dot-parser.test.ts",
+        "test/edges/bezier-edge-dynamic.test.ts",
+        "test/edges/bezier-edge-static.test.ts",
+        "test/edges/cubic-bezier-edge.test.ts",
+        "test/edges/edge-base.test.ts",
+        "test/edges/end-points.test.ts",
+        "test/edges/helpers.ts",
+        "test/edges/straight-edge.test.ts",
+        "test/gephi-parser.test.ts",
+        "test/helpers/index.ts"
+      ],
+      rules: {
+        "prettier/prettier": [
+          "error",
+          {
+            endOfLine: "lf",
+            parser: "typescript",
+            printWidth: 80,
+            quoteProps: "consistent",
+            semi: false,
+            singleQuote: true,
+            tabWidth: 2,
+            trailingComma: "es5",
+            useTabs: false
+          }
+        ],
+
+        // @TODO: Seems to mostly work just fine but I'm not 100 % sure.
+        // @TODO: Deprecated, anything like this for tsdoc?
+        "valid-jsdoc": [
+          "error",
+          {
+            prefer: {
+              arg: "param",
+              argument: "param",
+              return: "returns"
+            },
+            requireParamDescription: true,
+            requireParamType: false,
+            requireReturn: false, // Requires return for void functions.
+            requireReturnDescription: true,
+            requireReturnType: false
+          }
+        ],
+
+        // Class related.
+        "@typescript-eslint/member-naming": [
```

Comment:
Once more: Why do we need eslint if we already have prettier?

Reply:
Prettier is only about formatting. ESLint does a lot more than that (it reports unused variables, missing JSDoc etc.)